AI/libs/utils/news_processing_requests.py

```
@@ -0,0 +1,212 @@
+        page += 1 # 다음 페이지로
+
+    return pd.DataFrame({"href": href_list, "title": title_list, "date": time_list})
+
+def _news_content_crawl(url_list: List[str]) -> List[str]:
+    '''
+    news_href_crawl에서 수집된 뉴스 본문 수집 함수 (requests + bs4)
+    '''
+    content_list = []
+    for url in tqdm(url_list):
+        try:
+            response = requests.get(url, headers={'User-Agent': 'Mozilla/5.0'})
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)